apps/wing/project-templates/wing/empty/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "my-wing-app",
+  "version": "0.0.0",
+  "description": "A description of my Wing application",
+  "author": "Your Name",
+  "license": "MIT",
```

Comment:
Do you think we should add a `test`, `build` and `start`/`run`/`dev` scripts?

Reply:
I think teaching people how to use `wing` commands makes more sense to me to reinforce it's a dedicated tool

Reply:
OK, agreed.